src/wallet/rpcwallet.cpp

```diff
@@ -1444,13 +1444,19 @@ void ListTransactions(const CWalletTx& wtx, const string& strAccount, int nMinDe
                 entry.push_back(Pair("involvesWatchonly", true));
             entry.push_back(Pair("account", strSentAccount));
             MaybePushAddress(entry, s.destination);
-            std::map<std::string, std::string>::const_iterator it = wtx.mapValue.find("DS");
-            entry.push_back(Pair("category", (it != wtx.mapValue.end() && it->second == "1") ? "privatesend" : "send"));
+            if (s.vout >= 0) {
+                std::map<std::string, std::string>::const_iterator it = wtx.mapValue.find("DS");
+                entry.push_back(Pair("category", (it != wtx.mapValue.end() && it->second == "1") ? "privatesend" : "send"));
+            } else {
+                entry.push_back(Pair("category", "mixedsend"));
```

Comment:
`mixedsend`?... I guess it's smth to mark inputs from other participants i.e. external inputs, so why don't we just call them `external`?